src/warc2zim/converter.py

```diff
@@ -397,7 +397,12 @@ def gather_information_from_warc(self):
             if record.rec_type not in ("response", "revisit"):
                 continue
 
-            url = get_record_url(record)
+            url = str(get_record_url(record))
```

Comment:
`get_record_url()`  has no return type but can it ever return other than an `str`? Wouldn't it be better then to set the return type?

Reply:
Agreed